website/docs/docs/dbt-versions/core-upgrade/01-upgrading-to-v1.8.md

```diff
@@ -60,21 +60,30 @@ dbt test --select "test_type:data"           # run all data tests
 
 Unit tests are defined in YML files in your `models/` directory and are currently only supported on SQL models. To distinguish between the two, the `tests:` config has been renamed to `data_tests:`. Both are currently supported for backward compatibility.
 
-### The `--empty` flag
+#### New `data_tests:` syntax
 
-The [`run`](/reference/commands/run#the-`--empty`-flag) and [`build`](/reference/commands/build#the---empty-flag) commands now support the `--empty` flag for building schema-only dry runs. The `--empty` flag limits the refs and sources to zero rows. dbt will still execute the model SQL against the target data warehouse but will avoid expensive reads of input data. This validates dependencies and ensures your models will build properly.
+The `tests:` syntax is changing to reflect the addition of unit tests. Start migrating your [data test](/docs/build/data-tests#new-test-syntax) YML to use `data_tests:` after you upgrade to v1.8 to prevent issues in the future.
+
+```yml
+
+models:
+  - name: orders
+    columns:
+      - name: order_id
+        data_tests:
+          - unique
+          - not_null
 
-### Additional attributes in run_results.json
 
-The run_results.json now includes three attributes related to the `applied` state that complement `unique_id`:
+```
 
-- `compiled`: Boolean entry of the node compilation status (`False` after parsing, but `True` after compiling).
-- `compiled_code`: Rendered string of the code that was compiled (empty after parsing, but full string after compiling).
-- `relation_name`: The fully-qualified name of the object that was (or will be) created/updated within the database.
+### The `--empty` flag
 
+The [`run`](/reference/commands/run#the-`--empty`-flag) and [`build`](/reference/commands/build#the---empty-flag) commands now support the `--empty` flag for building schema-only dry runs. The `--empty` flag limits the refs and sources to zero rows. dbt will still execute the model SQL against the target data warehouse but will avoid expensive reads of input data. This validates dependencies and ensures your models will build properly.
```

Comment:
How do you know the dependencies have been validated?

Reply:
If the models build successfully. It's not much different than a normal `build` or `run` command except it limits the rows to zero.

Reply:
OK gotcha! So they see passing tests. That makes sense. Carry on...nothing to see here...I will approve